common/src/jsMain/kotlin/dev/kord/common/Platform.kt

```diff
@@ -0,0 +1,13 @@
+package dev.kord.common
+
+public actual object Platform {
+    public actual val IS_JVM: Boolean = false
+    public actual val IS_NODE: Boolean
+        get() = js(
+            "typeof process !== 'undefined' && process.versions != null && process.versions.node != null"
+        ) as Boolean
+    public actual val IS_BROWSER: Boolean
+        get() = js(
+            "typeof window !== 'undefined' && typeof window.document !== 'undefined' || typeof self !== 'undefined' && typeof self.location !== 'undefined'"
+        ) as Boolean
```

Comment:
i'm curious, where do you have this cursed logic from?

Reply:
https://github.com/ktorio/ktor/blob/23261519a5621a3401ab8b6db18740ad01f03b79/ktor-utils/js/src/io/ktor/util/PlatformUtilsJs.kt

Reply:
and the browser part?